Modules/@babylonjs/react-native/EngineHook.ts

```diff
@@ -185,6 +185,44 @@ declare var _native: {
     }
 }
 
+export function useAppState(): string {
```

Comment:
If we are exporting these, I guess we are saying they are part of the BRN public contract? Or should we not consider them part of the public contract and not re-export them from `index.js`?